server/lib/csp.js

```diff
@@ -13,10 +13,15 @@ var helmet = require('helmet');
 var config = require('./configuration');
 var url = require('url');
 
-var SELF = "'self'";
+var SELF = '\'self\'';
```

Comment:
Yeah, use of `'"'` is permitted but only for the purpose of avoiding the insanity of `'\'`.